rsrc/dialogs/edit-quest.xml

```diff
@@ -10,20 +10,21 @@
 	<text top='85' left='50' width='118' height='16'>Quest Description:</text>
 	<field name='name' top='57' left='205' width='252' height='16'/>
 	<field name='descr' top='84' left='205' width='252' height='104'/>
-	<text top='209' left='50' width='160' height='28'>Must be completed by day:<br/>(-1 for no deadline)</text>
+	<text top='209' left='50' width='170' height='28'>Must be completed by day:<br/>(-1 for no deadline)</text>
 	<field name='chop' top='208' left='205' width='110' height='16'/>
-	<text top='247' left='50' width='150' height='28'>Event to waive deadline:<br/>(-1 for none)</text>
-	<field name='evt' top='246' left='205' width='100' height='16'/>
+	<text top='247' left='50' width='170' height='28'>Event to waive deadline:<br/>(-1 for none)</text>
+	<field name='evt' top='246' left='205' width='60' height='16'/>
+	<button name='choose-evt' type='regular' anchor='evt' relative='pos pos-in' top='-4' left='8'>Choose</button>
 	<text size='large' top='280' left='50' width='220' height='17'>Reward for completing:</text>
 	<text top='307' left='50' width='150' height='16'>Experience Points:</text>
 	<field name='xp' top='306' left='205' width='110' height='16'/>
 	<text top='333' left='50' width='150' height='16'>Gold:</text>
 	<field name='gold' top='332' left='205' width='110' height='16'/>
-	<led name='rel' wrap='true' top='208' left='330'>Deadline is relative to start day</led>
-	<led name='start' wrap='true' top='246' left='330'>Player is given quest when scenario starts</led>
-	<led name='inbank' top='285' left='330'>Include in a job bank:</led>
-	<field name='bank1' top='306' left='344' width='110' height='16'/>
-	<field name='bank2' top='332' left='344' width='110' height='16'/>
+	<led name='rel' wrap='true' top='208' left='345'>Deadline is relative to start day</led>
```

Comment:
This could either mean the scenario start day, or quest start day. I'm not sure which it means.